kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -776,26 +797,30 @@ def test_duplicate_submission_with_partial_perms(self):
             PERM_PARTIAL_SUBMISSIONS,
             partial_perms=partial_perms,
         )
+        self.asset.assign_perm(
+            self.anotheruser,
+            PERM_ADD_SUBMISSIONS,
+        )
```

Comment:
Great, but out of curiosity: did this fail before the change?

Reply:
Let's verify that  'add_submissions' is automatically added when partial 'change_submissions' permission is assigned.

Reply:
`add_submissions` is indeed added automatically (in the next PR)
https://github.com/kobotoolbox/kpi/blob/a1358ea7bf237172edd439a6bb67ad2d1796fc8b/kpi/models/asset.py#L1153-L1161

The test was not failing because we were expecting `change_submissions` to duplicate a submission. 
It started to fail when we discussed that both permissions (`change_submissions` and `add_submissions`) were needed. 
Our final decision was to imply `add_submissions` when assigning `change_submissions`. 

These lines can be removed.